.circleci/config.yml

```diff
@@ -69,12 +67,11 @@ jobs:
             docker cp ~/st2-packages st2-packages-vol:/root
       - run:
           name: Pull dependent Docker Images
-          command: .circle/docker-compose2.sh pull ${DISTRO}
+          command: .circle/docker-compose2.sh pull ${DISTRO} || .circle/docker-compose2.sh pull ${DISTRO}
       - restore_cache:
           name: Restore Wheelhouse Cache
           keys:
-            - wheelhouse-v2-{{ .Branch }}-{{ checksum "/tmp/distro-version" }}-{{ checksum "/tmp/st2-version" }}
-            - wheelhouse-v2-{{ .Branch }}-
```

Comment:
this one is a potentially dangerous fallback as a cache produced by distro winning the first build race might be used then by all the future builds